test/texparser/tar_extractor_test.py

```diff
@@ -0,0 +1,57 @@
+import os
+import shutil
+import sys
+import unittest
+
+sys.path.append(".")
+from texparser.tar_extractor import TarExtractor
+
+
```

Comment:
for Unitttesting a nice framework for naming functions would be function name double underscore condition  underscore expectedResult:
example:
test_strip_special_chars__stringInputAndNoSpecialChars_returnStringInput:
functiontotest = test_strip_special_chars
condition = stringInputAndNoSpecialChars
expectedResult = returnStringInput

Reply:
A good Framework for writing tests and giving them structure would be the arrange act assert pattern:
detailed here:https://automationpanda.com/2020/07/07/arrange-act-assert-a-pattern-for-writing-good-tests/

just add 3 comments with #arrange #act #assert for structuring the tests

Reply:
We will ignore it for now, but add it to our conventions and clean it later.